### PR TITLE
Configurable SpriteSelectorItems per row. Standardized spacer variable. MenuBar Pass

### LINKED ISSUE #90: 5 SpritesSelectorItems per row
1. Fix `space-between` issue when either 2-3 sprite items are in a row [vs either 1, or 4]. This will also let us remove margin on the sprite item component, and add it to parent as a independent class.

2. Refactor sprites from set width to PostCSS `Lost grid` (maybe)

### PR DESCRIPTION
**Pull request by lifeinchords**
Resolves GH-90
Related GH-84
Related GH-67

### Changes
* SpriteSelectorItems: fixed spacing between rows. Isolated layout + component CSS, now reusable in other context. Items per row is now easily configurable
* Standardized use of space units where appropriate
* Reversed order of Box classes for easier readability in Inspector
* Sprite Area: removed unused info button
* MenuBar: refactored structure for clickable items

### CODE REVIEW

**Comment by lifeinchords** [comment]
screen shot 2017-02-23 at 7 21 38 pm

**Comment by thisandagain** [comment]
This looks good to me! Thanks @lifeinchords.

File: src/components/menu-bar/menu-bar.jsx [hunk 1]
```diff
@@ -12,14 +12,20 @@ const MenuBar = function MenuBar () {
...
             </div>
+            <div className={styles.menuItem} >Animation Playtest Prototype</div>
 
-            <span className={styles.title}>Animation Playtest Prototype</span>
+            {/*
+
+                <a className={styles.menuItem} href="#" >Load</a>
+                <a className={styles.menuItem} href="#" >Save</a>
+                <a className={styles.menuItem} href="#" >Open in Scratch</a>
+            */}
```
Comment (rschamp) [inline src/components/menu-bar/menu-bar.jsx]: Nit: remove commented out code

**Comment by lifeinchords** [comment]
@rschamp Refactored to simpler method, and cleaned things up. Please have a look

**Pull request merged**